contracts/Conference.sol

```diff
@@ -89,10 +91,19 @@ contract Conference is Destructible, GroupAdmin {
 	}
 
 	function register(string _participant) external payable onlyActive{
+		Logger(msg.sender, 'Starting register');
```

Comment:
Ultimately remove all Logger statements while not in use -- depending on the gas cost?